core/state_processor_rip7560.go

```diff
@@ -557,3 +596,18 @@ func validateValidityTimeRange(time uint64, validAfter uint64, validUntil uint64
 	}
 	return nil
 }
+
+func (vpr *ValidationPhaseResult) OnEnter(depth int, typ byte, from common.Address, to common.Address, input []byte, gas uint64, value *big.Int) {
+	if vpr.OnEnterSuper != nil {
+		vpr.OnEnterSuper(depth, typ, from, to, input, gas, value)
```

Comment:
This is not enough, since the "super" is not a method on "vpr", and the callbacks are not "bound methods" as in javascript.
 (which means this call-to-super is more complex. need to think if we want to enable it now)

Reply:
I actually think that in this case, as I assign `OnEnterSuper` to a method of an instance (`vpr.OnEnterSuper = evm.Config.Tracer.OnEnter`) the `rip7560ValidationTracer` remains "bound". It is a bit confusing but I checked manually and it does work.

Have a look:
https://go.dev/ref/spec#Method_values